src/main/java/io/temporal/internal/metrics/MetricsType.java

```diff
@@ -66,15 +60,15 @@
   public static final String ACTIVITY_E2E_LATENCY =
       TEMPORAL_METRICS_PREFIX + "activity_endtoend_latency";
   public static final String ACTIVITY_TASK_ERROR_COUNTER =
-      TEMPORAL_METRICS_PREFIX + "activity_task_error";
+      TEMPORAL_METRICS_PREFIX + "activity_task_error_thrown";
```

Comment:
can we just call it activity_task_error? _thrown suffix won't be meaningful for go-sdk.